R/config.R

```diff
@@ -292,14 +348,7 @@ validate_config_task_id_text <- function(predevals_config, task_groups, task_id_
   for (i in seq_along(predevals_config$task_id_text)) {
     task_id_text_item <- predevals_config$task_id_text[[i]]
     task_id_name <- names(predevals_config$task_id_text)[i]
-    hub_task_id_values <- purrr::map(
-      task_groups,
-      function(task_group) {
-        task_group$task_ids[[task_id_name]]
-      }
-    ) |>
-      unlist() |>
-      unique()
+    hub_task_id_values <- get_task_id_values(task_groups, task_id_name)
```

Comment:
Refactored lines 295 - 302 into the new `get_task_id_values` function in R/utils-hub_tasks_config.R`

Reply:
this seems like a function that would be appropriate to have in hubUtils.